packages/cursorless-org-docs/src/docs/user/release-notes/0.29.0.md

```diff
@@ -0,0 +1,87 @@
+
+### Cursorless everywhere
+
+We're taking those first wobbly, beautiful steps out of VSCode 🐣🥲. We have now shipped very early, experimental support for [Cursorless in Neovim](https://github.com/hands-free-vim/cursorless.nvim). It's still missing hats and a parse tree, but the engine is there. We've landed; now we'll expand. Big shout-out to the dream team, [@saidelike](https://github.com/saidelike) and [@fidgetingbits](https://github.com/fidgetingbits), for the long hours and shower of tears as this came together. This one shipped with continuous delivery, because we're psycho about CD, so any improvements to Cursorless will automatically make their way into Neovim using the confusingly-named Lazy package manager 🤔
+
+We also have an internal experiment with a version of Cursorless that runs literally anywhere, using accessibility APIs. This one is so experimental that I'll probably get in trouble for even mentioning it. So definitely don't heckle @AndreasArvidsson and @phillco to tell you how to play with this completely not-ready-for-primetime early experiment.
+
+### New hat shapes
+
+It's unfortunate that we don't record our Discord sessions, because the discussions about the new hat shapes would make fantastic examples for the Wikipedia page on [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality). I'm really happy with how they turned out. Except for fox. Such a shame about that pixel.
+
+### Strange Loop 2023
+
+ICYMI Cursorless went to Strange Loop. The talk was so wild that they had to close Strange Loop forever. Sorry. I really liked Strange Loop too. Here's the talk ([video 🎬](https://youtu.be/NcUJnmBqHTY)); I think it's a nice way to understand the concepts underlying Cursorless. Its Platonic form, if you will. And I really hope that you will.
+
+### Improved Talon-side API
+
+We beefed up our Talon-side API to allow you to build your own grammars on top of Cursorless. See the [talon-side api docs](https://www.cursorless.org/docs/user/customization/#public-talon-actions) for more.
+
+We also have an undocumented, secret action that has a built-in parser for building arbitrarily powerful custom commands on the fly. You should also definitely not heckle @AndreasArvidsson and @phillco to tell you how to use this one either.
+
+### Experimental keyboard interface
+
+The keyboard interface continues to grow and is getting close to feature-parity with our spoken grammar, but is still very much in the experimental phase. We're looking for feedback on this one, so if you're interested in trying it out, come find us on Slack or Discord.
+
+### Falling back
+
+You know when you're in a text box or the VSCode integrated terminal and you're like "Man, I wish I could bring that line right to my cursor"? Do it. Go on. `"bring line air"`. You can also use some hatless Cursorless commands on your cursor in that text box (`"take token"`, `"copy two tokens backward"`, etc etc etc 🙄)
```

Comment:
Why the eye roll?

Reply:
Just showing off about how many different things you can do. Kind of like "yeah whatever no big deal"

Reply:
Ah, I think 😅 or 😎 would capture that intent better